codec-http/src/main/java/io/netty/handler/codec/http/HttpContentCompressor.java

```diff
@@ -212,4 +241,54 @@ protected ZlibWrapper determineWrapper(String acceptEncoding) {
         }
         return null;
     }
+
+    @SuppressWarnings("FloatingPointEquality")
+    protected String determineEncoding(String acceptEncoding) {
```

Comment:
Implement the other `determineEncoding` in terms of this method, to avoid the logic duplication.